spec/requests/api/internal/v1/prosecution_cases_spec.rb

```diff
@@ -40,17 +40,21 @@
                     description: "Searches prosecution cases by prosecution case reference"
 
           parameter name: "include", in: :query, required: false, type: :string,
-                    schema: {},
-                    description: 'Return defendant, offence and hearing data through a has_many relationship </br>
-                                  eg include=defendants,defendants.offences,defendants.defence_organisation,hearings.providers,hearing_summaries'
+                    schema: {
```

Comment:
Like I said in the other PR:

Elsewhere in this file, there is inconsistency in naming the search variable names as part of the description. I wonder if this PR would be a good time to tidy them.

`description: "Searches prosecution cases by national_insurance_number"
`vs
`description: "Searches prosecution cases by arrest summons number"`

My preference would be to not have the underscores, but I don't mind as long as they're all consistent